lib/src/widgets/chat/ds_end_calls_message_bubble.widget.dart

```diff
@@ -0,0 +1,272 @@
+
+  bool get _isInbound =>
+      content['direction'].toString().toLowerCase() == 'inbound';
+
+  bool get _isLightBubbleBackground => style.isLightBubbleBackground(align);
+  bool get _isDefaultBubbleColors => style.isDefaultBubbleBackground(align);
+
+  Color get _foregroundColor => _isLightBubbleBackground
+      ? DSColors.neutralDarkCity
+      : DSColors.neutralLightSnow;
+
+  DSEndCallsMessageBubble({
+    super.key,
+    required this.align,
+    required this.content,
+    required this.onAsyncFetchSession,
+    this.borderRadius = const [DSBorderRadius.all],
+    DSMessageBubbleStyle? style,
+    this.translations,
+  }) : style = style ?? DSMessageBubbleStyle();
+
+  @override
+  Widget build(BuildContext context) {
+    return DSMessageBubble(
+      borderRadius: borderRadius,
+      align: align,
+      style: style,
+      child: Column(
+        crossAxisAlignment: CrossAxisAlignment.start,
+        children: [
+          Row(
+            mainAxisAlignment: MainAxisAlignment.spaceBetween,
+            crossAxisAlignment: CrossAxisAlignment.start,
+            children: [
+              Column(
+                children: [
+                  Row(
+                    children: [
+                      Padding(
+                        padding: const EdgeInsets.only(right: 8.0),
+                        child: Column(
+                          children: [
+                            Container(
+                              decoration: BoxDecoration(
+                                color: _isCallAnswered
+                                    ? DSColors.success
+                                    : DSColors.error,
+                                borderRadius: const BorderRadius.all(
+                                  Radius.circular(
+                                    8.0,
+                                  ),
+                                ),
+                              ),
+                              width: 30,
+                              height: 30,
+                              child: Icon(
+                                _isCallAnswered
+                                    ? _isInbound
+                                        ? DSIcons.voip_receiving_outline
+                                        : DSIcons.voip_calling_outline
+                                    : DSIcons.voip_ended_outline,
+                                size: 20.0,
+                              ),
+                            ),
+                          ],
+                        ),
+                      ),
+                      Column(
+                        crossAxisAlignment: CrossAxisAlignment.start,
+                        children: [
+                          DSCaptionText(
+                            'calls.voice-text'.translate(),
+                            fontWeight: FontWeight.bold,
+                            color: _foregroundColor,
+                          ),
+                          DSCaptionSmallText(
```

Comment:
According to prototype, this text style must be `DSCaptionText` with `fontWeight: DSFontWeights.semiBold`.